WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -255,6 +255,11 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         return 0.0
     }
 
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let page = pageAtIndexPath(indexPath)
+        return page.canDisplayTags ? 60.0 : 44.0
```

Comment:
Could these values be constants like `pageSectionHeaderHeight` and `pageCellEstimatedRowHeight`?  Actually, we probably want to use `pageCellEstimatedRowHeight` which has a height of `47.0` vs the `44.0` here.